### DISCUSSION TITLE
Renovatebot does not understand `# tag=v3` [misleading title]

### DISCUSSION THREAD

**Discussion by laurentsimon** [renovatebot/renovate#16765] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I added tag to migrate from dependabot to renovatebot `# tag=vX`. renovatebot is unable to display which version the PR is updating to in the description of the PR (https://github.com/slsa-framework/slsa-verifier/pull/165).

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by rarkins** [comment, marked as answer]
You have chosen to use a major tag instead of a major.minor.patch. Therefore Renovate is not updating a version and instead is updating a SHA. It corresponds to https://github.com/actions/checkout/releases/tag/v3

If you want to know which version it is then use e.g. `# tag=v3.0.2` and then you'll see the version and hash upgrading in sync

**Reply by rarkins** [reply]
To clarify: Renovate exactly understands `v3`, and has updated to the latest SHA for that tag. There is no bug, at best this just a matter of opinion on a cosmetic detail

**Reply by laurentsimon** [reply]
Gotcha. It'd be nice for the PR to indicate which version the new hash corresponds to. In my 2-week experience with renovatebot: when it only indicates a hash in the PR, it's often indicative that we forgot to add `# tag=X` in the workflow. So it's confusing to me / us.

**Reply by rarkins** [reply]
As I mentioned above you will see this if you use an exact version in the tag, e.g. `v3.0.2`. There is not even a guarantee that a `v3` tag will have a corresponding `v3.x.y` tag with same SHA (although it's usually the case)
